molecule/docker/molecule.yml

```diff
@@ -0,0 +1,33 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+lint:
+  name: yamllint
+platforms:
+  - name: docker
+    image: replicajune/ubuntu-dind-supervisord
+    docker_host: "${DOCKER_HOST:-unix://var/run/docker.sock}"
+    privileged: true
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+      - /var/run/docker.sock:/var/run/docker.sock:rw
+      - /root/.ansible:/root/.ansible:rw
+      - /home:/home:rw
+provisioner:
+  name: ansible
+  lint:
+    name: ansible-lint
+  playbooks:
+    create: create.yml
```

Comment:
This should link to files from `default/`. Look at https://github.com/cloudalchemy/ansible-prometheus/blob/189ab57c67359cebfdbb4c951085f2c083d59f51/molecule/alternative/molecule.yml#L50-L53

Reply:
I've used soft links on couple of repository when I needed the feature but not available with the tool I worked with, nice to see it's built in with molecule !